docs/configs/frontera.py

```diff
@@ -21,22 +20,21 @@
             max_workers_per_node=2,
             address=address_by_hostname(),
             provider=SlurmProvider(
-                cmd_timeout=60,     # Add extra time for slow scheduler responses
+                cmd_timeout=60,  # Add extra time for slow scheduler responses
                 nodes_per_block=2,
                 init_blocks=1,
                 min_blocks=0,
                 max_blocks=1,
-                partition=user_opts['frontera']['partition'],  # Replace with partition name
-
+                partition=user_opts["frontera"][
+                    "partition"
```

Comment:
This looks a bit ugly?

Reply:
Per our conversation, I reverted the `doc/configs/` to the content in `main` and added `fmt: off` on them.

I think if we _had to_ we could find a way to write this which is user-friendly and has `black` applied, but it's not necessary -- at least for now.